src/Utility/ExprFunctions.C

```diff
@@ -234,31 +213,7 @@ EvalFunction::EvalFunction (const char* function, Real epsX, Real epsT)
 }
 
 
-EvalFunction::~EvalFunction ()
-{
-  this->cleanup();
-}
-
-
-void EvalFunction::cleanup ()
-{
-  for (Expression* it : expr)
-    delete it;
-  for (FunctionList* it : f)
-    delete it;
-  for (ValueList* it : v)
-    delete it;
-  for (EvalFunction* it : gradient)
-    delete it;
-  for (EvalFunction* it : dgradient)
-    delete it;
-  expr.clear();
-  f.clear();
-  v.clear();
-  arg.clear();
-  gradient.fill(nullptr);
-  dgradient.fill(nullptr);
-}
+EvalFunction::~EvalFunction () = default;
```

Comment:
Iinline default destructor in header file instead

Reply:
see above.